api/java/org/openyolo/api/CredentialClient.java

```diff
@@ -272,7 +296,15 @@ public CredentialRetrieveResult getCredentialRetrieveResult(
         }
 
         try {
-            return CredentialRetrieveResult.fromProtobufBytes(resultBytes);
+            CredentialRetrieveResult result =
+                    CredentialRetrieveResult.fromProtobufBytes(resultBytes);
+
+            // Once a successfully retrieve result has been handled, re-enable auto sign-in.
```

Comment:
I think I am in favor of until a successful response is returned as it should guarantee that the user consented to an auto sign in account at least once before being automatically signed in. Consider the case: signout, disableAutoSignIn() > retrieve w/ requireUserMediation=true, but user cancels > retrieve w/ requireUserMediation=false, auto signed in.  The user can now be auto signed in without specifying which account they want to use.